player/command.c

```diff
@@ -6966,8 +6966,11 @@ void mp_option_change_callback(void *ctx, struct m_config_option *co, int flags,
     bool init = !co;
     void *opt_ptr = init ? NULL : co->data; // NULL on start
 
-    if (co)
+    if (co) {
         mp_notify_property(mpctx, co->name);
+        if (opt_ptr == &opts->media_title)
+            mp_notify_property(mpctx, "media-title");
```

Comment:
I don't think there's a need for this to be in the `if (co)`, is there?
all the other checks are a bunch of lines below

also not sure if it would be cleaner to do `mp_notify(mpctx, MP_EVENT_METADATA_UPDATE, NULL)`

Reply:
At first I had it at the end of the function, but then I though it should probably be next to the other notify call.

The actual metadata of the file doesn't change when `force-media-title` is set, so I don't think `mp_notify(mpctx, MP_EVENT_METADATA_UPDATE, NULL)` makes sense here. It would work but seems overkill.

Reply:
Actually `MP_EVENT_METADATA_UPDATE` is better because of the playlist title thing, but I could factor that out into it's own function and call that.

Whichever you prefer.

Reply:
not sure what you mean.
just put the right `mp_notify` in this function and call it a day?

Reply:
This is what I mean.

Reply:
I've changed it to the `mp_notify` for simplicity, it's not like that will cause any performance problems.

Reply:
> I've changed it to `mp_notify` for simplicity, it's not like that will cause any performance problems.